jacobalberty/unifi-docker
# Add gitignore for backup folder

## Description

Add a gitignore at the root folder to ignore backup folder when the stack is launched.

## Motivation and Context

By default, backup folder is created at the root folder based on docker-compose.yml

![image](https://user-images.githubusercontent.com/31205250/151693825-af14070f-6368-4e2f-998f-177e3b05286f.png)

When it's done, git is saying that backup folder is unfollow for now.

![image](https://user-images.githubusercontent.com/31205250/151693815-008ba53e-578d-4012-9728-0180bccb4b4d.png)

This PR is just to make this thing "prettier", nothing more.

## How Has This Been Tested?

- Add a gitignore
- Check how git reacts.
